test/common/testlib.py

```diff
@@ -717,7 +728,7 @@ def click_system_menu(self, path, enter=True):
         (useful for remote hosts).
         '''
         self.switch_to_top()
-        if self.cdp.mobile:
+        if self.current_layout["is_mobile"] == "mobile":
```

Comment:
Typo, this is a bool.

Reply:
Ouch, and it doesn't seem to make any difference... evidence that this condition never needs to be true.